tests/php/test-class-amp-options-rest-controller.php

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
Depending on the outcome of the REST API controller above, this may have to be refactored.